packages/frontend/src/components/CreateUserForm/VerifyEmailForm.tsx

```diff
@@ -0,0 +1,75 @@
+import { Intent } from '@blueprintjs/core';
+import React, { FC, useState } from 'react';
+import Countdown, { zeroPad } from 'react-countdown';
+import { useForm } from 'react-hook-form';
+import {
+    AnchorLinkWrapper,
+    ErrorMessage,
+    Subtitle,
+} from '../../pages/SignUp.styles';
+import AnchorLink from '../common/AnchorLink';
+import Form from '../ReactHookForm/Form';
+import { PasswordInputField, SubmitButton } from './CreateUserForm.styles';
+
+export const VerifyEmailForm: FC<{
+    numberOfAttempts: number;
+    expirationTime: string;
+    onSubmit: (code: string) => void;
+    onResend: () => void;
+}> = ({ numberOfAttempts, expirationTime, onSubmit, onResend }) => {
+    const methods = useForm({
+        mode: 'onSubmit',
+        defaultValues: {
+            code: '',
+        },
+    });
```

Comment:
We should move this to the parent component that is also making the submit request.
In there we can set the input error message depending on the backend response. 
eg:

```
const methods = useForm({ mode: 'onSubmit' });
const { mutate, error } = useEmailVerificationMutation();
const {setError, clearErrors} = methods;
useEffect(() => {
    if(error) {
        const message = error.attempts < 5 ? 'The code doesn't match the one we sent you.' : 'Hmm that code doesn't match the one we sent you. You've already had 5 attempts, please resend a verification email and try again.';
        setError('code', { type: 'custom', message });
    } else {
        clearErrors('code');
    }
}, [error, clearErrors, setError])

```

This also covers @IrakliJani feedback on the custom error component.